docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -63,216 +37,137 @@ $ # Log out and log back in again to pick up the group changes
 $ exit
 ```
 
-#### Download and unpack Alluxio
+## Launch Alluxio
 
-1. [Download](https://www.alluxio.org/download) Alluxio and copy it to the linux machine.
-1. Unpack the Alluxio tarball to a directory.
 ```bash
-$ tar xvfz alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-<hadoop distribution>.tar.gz
+# Launch the Alluxio master
+$ docker run -d --net=host
+    -v /mnt/data:/opt/alluxio/underFSStorage
+    alluxio/alluxio master
+# Launch the Alluxio worker
+$ docker run -d --net=host
+    --shm-size=1G -e ALLUXIO_WORKER_MEMORY_SIZE=1G
+    -v /mnt/data:/opt/alluxio/underFSStorage
+    -e ALLUXIO_MASTER_HOSTNAME=localhost
+    alluxio/alluxio worker
 ```
 
-#### Build the Alluxio Docker image
-
-```bash
-$ cd alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-<hadoop distribution>/integration/docker
-$ mv ../../../alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-<hadoop distribution>.tar.gz .
-$ docker build -t alluxio --build-arg ALLUXIO_TARBALL=alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-<hadoop distribution>.tar.gz .
-```
+These commands use the host machine's `/mnt/data` directory as the under storage for Alluxio. 
+The `--shm-size=1G` argument will allocate a `1G` tmpfs for the worker to store Alluxio data.
 
-#### Set up under storage
+## Verify the Cluster
 
-Create an under storage folder on the host.
+To verify that the services came up, check `docker ps`. You should see something like
 ```bash
-$ mkdir underStorage
+$ docker ps
+CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS               NAMES
+ef2f3b5be1a3        alluxio:1.8.0       "/entrypoint.sh work…"   6 days ago          Up 6 days                               dazzling_lichterman
```

Comment:
nit: this is different from the image you asked them to use. change to alluxio/alluxio:1.8.0?

Reply:
good catch, updated